CONTRIBUTING.md

```diff
@@ -14,9 +14,10 @@ Once you've cloned the repository to your local machine, follow these steps to g
 
 1. Install the development dependencies by running the following command: `npm install`
 2. Configure a `foundryconfig.json` file in the root folder of the project with a `dataPath` property. And example can be found in the `foundryconfig.example.json` file provided in the project. Just copy the file and rename it to `foundryconfig.json` and change the `dataPath`. The `dataPath` property is the file path to your Foundry installs user data. This can be found on the Configuration tab on the Setup screen.
-3. Run `npm run build`. This will compile all of the `less` and copy all of the necessary files to a `dist` folder in the project root. It will also create a symbolic link to this new folder with your Foundry installs user data path.
+3. If you already have the Starfinder RPG System installed on Foundry VTT, uninstall from Foundry it so that the next steps will properly set up your Symlink.
```

Comment:
"uninstall from Foundry it so..." doesn't make any sense. I'm assuming that you meant "uninstall it from Foundry so...".

Reply:
You're 100% right. Thanks for catching that!